quiche/src/lib.rs

```diff
@@ -8068,8 +8068,12 @@ impl<T> UnknownTransportParameter<T> {
     ///
     /// See Section 18.1 in [RFC9000](https://datatracker.ietf.org/doc/html/rfc9000#name-reserved-transport-paramete).
     pub fn is_reserved(&self) -> bool {
-        let n = (self.id - 27) / 31;
-        self.id == 31 * n + 27
+        if self.id < 27 {
+            false
+        } else {
+            let n = (self.id - 27) / 31;
```

Comment:
A simpler fix would be to have something like:
```suggestion
            let n = self.id.saturating_sub(27) / 31;
```
to avoid the if / else.

Reply:
Might be worth adding explicit test cases for `0` and `27` TPs, just to make sure we cover the edge cases.